app/controllers/stages_controller.rb

```diff
@@ -106,7 +106,11 @@ def clone
   private
 
   def badge_safe(string)
-    CGI.escape(string.gsub('&', '&amp;')).gsub('-', '&mdash;').gsub('+','%20')
+    CGI.escape(string.gsub('&', '&amp;'))
+      .gsub('+','%20')
+      .gsub /([^-])(--)*-([^-])/ do |r|
```

Comment:
http://rubular.com/r/mQqDnMGSk4

Not sure - Rubular seems to think it will insert `--` between every char?